src/it/test11-prefixes/verify.groovy

```diff
@@ -0,0 +1,11 @@
+
+def prefixes ( ) {
+	Process proc = ('rpm --qf "[%{Prefixes}\n]" -qp ' + basedir + "/target/test11.rpm").execute();
```

Comment:
For me the integration tests failed due with an empty prefix. Looks like the `\n` caused some unexpected response. I was able to fix this by using colon (:) as a delimiter:

~~~groovy
def prefixes ( ) {
	Process proc = ('rpm --qf [%{Prefixes}:] -qp ' + basedir + "/target/test11.rpm").execute();
	return proc.in.getText().trim();
}

def result = prefixes();
println "Prefixes: " + result;


return result ==  "/opt:/var/log:";
~~~